api/serverless.yml

```diff
@@ -330,7 +330,7 @@ functions:
         handler: src/components/coauthor/routes.sendApprovalReminder
         events:
             - http:
-                  path: ${self:custom.versions.v1}/publications/{id}/coauthors/{coauthor}/approval-reminder
+                  path: ${self:custom.versions.v1}/publications/{id}/coauthors/{coAuthor}/approval-reminder
```

Comment:
i remember these kind of changes can break the deploy due to some lambda limitations. Might worth testing the deploy still works with this change